public/locales/en-US.json

```diff
@@ -163,6 +176,8 @@
     "legendTooltip_Storage-gibibyte-months": "Data storage usage",
     "legendTooltip_threshold_Cores_OpenShift Container Platform": "Maximum capacity, as CPU cores, based on total {{product}} Annual subscriptions in this account.",
     "legendTooltip_threshold_Cores_rosa": "Maximum capacity, as vCPU hours, based on total ROSA Hosted Control Planes pre-paid subscriptions in this account.",
+    "legendTooltip_threshold_Managed-nodes": "Maximum capacity, based on total Ansible managed nodes pre-paid subscriptions in this account.",
+    "legendTooltip_threshold_Instance-hours_ansible-aap-managed": "Maximum capacity, as infrastructure hours, based on total Ansible managed nodes pre-paid subscriptions in this account.",
```

Comment:
Maximum capacity, as infrastructure hours, derived from the amount of managed nodes purchased in all subscriptions in this account.
NOTE: I *think* this is correct, but this should be confirmed because I haven't been able to get a confirmation yet. For example, if you prepay for 100 managed nodes a month, you get the amount of nodes x 24 x 30 as your capacity for infra.

Reply:
Based on this confirmation and other comments @jlprevatt I'll hold on updating until tomorrow

Reply:
Continuing updates, we'll iterate